tests/helpers/common/Chocolatey/Invoke-Choco.ps1

```diff
@@ -33,28 +33,43 @@
 
             $remainingArguments
         )
-
     }
     end {
+        $stopwatch = [System.Diagnostics.Stopwatch]::new()
+        $stopwatch.Start()
+
         $output = if ($PipelineInput) {
             $PipelineInput | & $chocoPath @arguments
         }
         else {
             & $chocoPath @arguments
         }
 
+        $stopwatch.Stop()
+        # We do not use env:ChocolateyInstall here as it is not guaranteed due to snapshots.
+        # Saving into the default Chocolatey install location as .log so Test Kitchen can pick up the file.
+        [PSCustomObject]@{
+            Date       = Get-Date -Format 'o'
+            Duration   = $stopwatch.Elapsed
+            Invocation = ($MyInvocation.PositionMessage -split "`r`n")[0]
+            Arguments  = ($Arguments -join ' ') -replace "`r`n", ''

```

Comment:
If we are to store the arguments, we need to remove/replace any sensitive arguments as well before they are stored on disk.

Reply:
I've removed the storing and returning of arguments for now as I don't currently have a reasonable way to ensure we're filtering out the appropriate things. Will possibly explore adding this in the future.